### PR TITLE
Use `--generate-link-to-definition` option when generating documentation

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: glib/patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
+diff --git a/glib/tests/subclass_compiletest/03-object-no-auto-send-sync.stderr b/glib/tests/subclass_compiletest/03-object-no-auto-send-sync.stderr
```
Comment (sdroege) [inline glib/patch:1]: What's this? :P

**Review by sdroege — DISMISSED** [review]
Looks good to me otherwise

File: glib/patch (hunk 1 above)
Comment (GuillaumeGomez) [reply glib/patch:1]: Oh no :o

**Comment by GuillaumeGomez** [comment]
Removed the patch file. ^^'

**Pull request merged**